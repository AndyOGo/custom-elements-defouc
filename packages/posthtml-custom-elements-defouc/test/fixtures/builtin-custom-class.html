<span is="builtin-element">content</span>
