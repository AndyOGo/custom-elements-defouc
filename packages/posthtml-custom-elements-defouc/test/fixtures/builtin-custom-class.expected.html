<span is="builtin-element" class="custom-elements-defouc">content</span>
